pkgs/by-name/li/librepods/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  kdePackages,
+  qt6,
+  pulseaudio,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "librepods";
+  version = "0.1.0-rc.4-unstable-2025-09-02";
+
+  src = fetchFromGitHub {
+    owner = "kavishdevar";
+    repo = finalAttrs.pname;
+    rev = "f547cc13c0cb1a021f6ea7b88056b1a54d440ebf";
+    hash = "sha256-3iqnHuZC16WWYauh/y+CuMkPdqmSNM5huQda7l1tW84=";
```

```suggestion
    hash = "";
```
will obviously need to be regenerated